### PR TITLE
fix(config): fix typos in `Vue` configuration files

### PR DESCRIPTION
**Pull request by 4ndu-7h4k**
# fix(config): fix typos in `Vue` configuration files

| ⏱️ Estimate | 📊 Priority | 📏 Size | 📅 Start | 📅 End |
| --- | --- | --- | --- | --- |
| 1h | P3 | XS | 08-10-2023 | 09-05-2026 |

## 🔄 Type of Change

- [x] Bug fix
- [ ] Breaking change
- [ ] Dependency
- [ ] New feature
- [ ] Improvement
- [x] Configuration
- [ ] Documentation
- [ ] CI/CD

## 📝 Summary

Fix typos in code comments across Vue configuration files and app component.

## 📋 Changes Made

- Fix "dont" to "don't" in CSS transition comment in `App.vue`
- Fix "rultes" to "rules" and "proccessing" to "processing" in `vue.config.js`
- Fix missing newline at end of `.vscode/settings.json`

## 🧪 Tests

- [x] Test Manual testing performed

### Files to modify
- `.vscode/settings.json`
- `src/App.vue`
- `vue.config.js`

### CODE REVIEW

**Comment by beatrizsmerino** [comment]
Thanks a lot for the corrections @4ndu-7h4k , now I'll take a look at them and tag this PR as part of the `hacktoberfest`.

File: src/App.vue
Comment (beatrizsmerino) [inline src/App.vue:1]: I agree with these changes

File: vue.config.js
Comment (beatrizsmerino) [inline vue.config.js:1]: I agree with these changes

File: .vscode/settings.json
Comment (beatrizsmerino) [inline .vscode/settings.json:1]: In this file I see many edited lines. The code has been formatted differently and I can't tell where the correction has been made.

**Review by beatrizsmerino — COMMENTED** [review]
Why did you delete the files? Did they have an error?

File: .vscode/extensions.json
Comment (beatrizsmerino) [inline .vscode/extensions.json:1]: Why did you delete the files? Did they have an error?
I want them to continue to exist

File: .vscode/settings.json
Comment (beatrizsmerino) [reply .vscode/settings.json:1]: Why did you delete the files? Did they have an error?
I want them to continue to exist

Comment (4ndu-7h4k) [reply .vscode/settings.json:1]: Deleted .vscode folder in line with .gitignore rules; now rectifying by restoring it.

**Review by beatrizsmerino — APPROVED** [review]

File: .vscode/settings.json
Comment (beatrizsmerino) [reply .vscode/settings.json:1]: Thanks a lot! Now it is merged
